exporter/elasticexporter/exporter.go

```diff
@@ -125,7 +125,10 @@ func newTransport(config *Config) (transport.Transport, error) {
 // returning the number of spans that were dropped along with any errors.
 func (e *elasticExporter) ExportResourceSpans(ctx context.Context, rs ptrace.ResourceSpans) (int, error) {
 	var w fastjson.Writer
-	elastic.EncodeResourceMetadata(rs.Resource(), &w)
+	if err := elastic.EncodeResourceMetadata(rs.Resource(), &w); err != nil {
+		return 1, err
```

Comment:
Is it possible to return the number of spans in `rs` that are being dropped when this error occurs?  Feels like it could be more than 1.

Reply:
Yes, you are right, I missed the comments when I fix this code